terraform/docs/applying-terraform.md

```diff
@@ -5,6 +5,20 @@ The EKS cluster is deployed via Terraform in two stages. See [adr-3] for backgro
 - `cluster-infrastructure` is concerned only with setting-up the EKS cluster and associated AWS resources (such as the worker groups and auto-scaling groups).
 - `cluster-services` is concerned only with setting up the Kubernetes resources and configuration for base services, including the `aws-auth` ConfigMap, ingress controller, etc.
 
+## Prerequisites
+
+1. Some of the deployments below require external [tfvars
+files](../terraform/deployments/variables). To create a new environment, you
+will need to copy the variables directory for an existing environment (e.g.
+[integration](../deployments/variables/integration)) and modify as
```

Comment:
i wonder if we should mention to copy name it to the name of the new environment, such as TEST etc...?

Reply:
I think this is what is saying, copy the directory